static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.ts

```diff
@@ -59,8 +56,20 @@ const config = _config as {
 	) => void;
 };
 
+interface TCFv2ConsentStateMockType {
+	consents: Record<string, boolean | null>;
+	vendorConsents: Record<string, boolean | null>;
+}
+
 const onConsentChange = onConsentChange_ as jest.MockedFunction<
-	(fn: any) => void
+	(
+		fn: (
+			arg0:
+				| { tcfv2: TCFv2ConsentStateMockType }
+				| { ccpa: CCPAConsentState }
+				| { aus: AUSRejectedMockType },
```

Comment:
I think this can be the `ConsentState` type from the CMP library.

Reply:
The mocked version of the types are here for tfcv2 and aus since the mocked return values in these tests don't implement all of the properties of ConsentState but just enough to test the implementation of the functions under test. 

Using these mocked types saves us implementing all the properties for each kind of ConsentState.